src/main/java/com/example/dog/dao/JdbcConnectionHolder.java

```diff
@@ -4,6 +4,9 @@
 import java.sql.Connection;
 import java.sql.SQLException;
 
+/**
+ * Was replaced with org.springframework.jdbc.datasource.DataSourceUtils
```

Comment:
Use `{@link org.springframework.jdbc.datasource.DataSourceUtils}` to reference classes/fields/methods